android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -224,6 +252,13 @@ class XMTPModule : Module() {
             }
         }
 
+        Function("getLibXMTPClientAccountAddress") { clientAddress: String -> String
+            logV("getLibXMTPClientAccountAddress")
+            val client = clients[clientAddress] ?: throw XMTPException("No client")
+            val libXMTPClient = client.libXMTPClient ?: throw XMTPException("No libxmtp client")
+            return@Function libXMTPClient.accountAddress()
+        }
```

Comment:
You don't need this is handled on the backend now. Should be the same as the address that gets returned from the client.

Reply:
fixed here 573b2afd0e07b64aefed47897c260e8a008d47d4